parameter_store/models.py

```diff
@@ -261,23 +256,7 @@ class Meta:
         max_length=128,
         null=True,
         blank=True,
-        help_text="This is an RFC 5545 recurrence rule, ex: FREQ=WEEKLY;BYDAY=WE,TH,FR"
-    )
-    maintenance_exclusion_name = models.CharField(
-        null=True,
-        blank=True,
-        max_length=64
-    )
-    maintenance_exclusion_start_1 = models.DateTimeField(
-        null=True,
-        blank=True,
-        help_text=None
-    )
-    maintenance_exclusion_end_1 = models.DateTimeField(
-        null=True,
-        blank=True,
-        help_text=None
-    )
```

Comment:
Can you help me understand why these were removed?

They were named after the repo SoT: https://github.com/GDC-ConsumerEdge/automated-cluster-provisioner/blob/main/example-source-of-truth.csv